docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -81,10 +81,7 @@
     "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    print("Creating objective function...")
</pre><p>Minor, optional nit: Can we add one more print statement at the end of the cell to just validate the cell executed fully? Otherwise it may look like it's still 'creating' fwiw</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/1132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
done.